bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/example_cards/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8/HToZATo2L2B_500p00_300p00_1p50_ggH_TuneCP5_13TeV_pythia8_run_card.dat

```diff
@@ -40,43 +40,46 @@
 # PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
 #*********************************************************************
   lhapdf	= pdlabel ! PDF set                                     
-$DEFAULT_PDF_SETS = lhaid ! if pdlabel=lhapdf, this is the lhapdf number
+325500 = lhaid ! if pdlabel=lhapdf, this is the lhapdf number
 #*********************************************************************
 # To see heavy ion options: type "update ion_pdf"
 #*********************************************************************
 # Renormalization and factorization scales                           *
 #*********************************************************************
-  False	= fixed_ren_scale ! if .true. use fixed ren scale
-  False	= fixed_fac_scale ! if .true. use fixed fac scale
-  91.188	= scale ! fixed ren scale
-  91.188	= dsqrt_q2fact1 ! fixed fact scale for pdf1
-  91.188	= dsqrt_q2fact2 ! fixed fact scale for pdf2
-  -1	= dynamical_scale_choice ! Choose one of the preselected dynamical choices
-  1.0	= scalefact ! scale factor for event-by-event scales
+  False = fixed_ren_scale ! if .true. use fixed ren scale
+  False = fixed_fac_scale ! if .true. use fixed fac scale
+  91.188    = scale ! fixed ren scale
+  91.188    = dsqrt_q2fact1 ! fixed fact scale for pdf1
+  91.188    = dsqrt_q2fact2 ! fixed fact scale for pdf2
+  -1    = dynamical_scale_choice ! Choose one of the preselected dynamical choices
+  1.0   = scalefact ! scale factor for event-by-event scales
 #*********************************************************************
 # Type and output format
 #*********************************************************************
-  False	= gridpack !True = setting up the grid pack
-  -1.0	= time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
-  average	= event_norm ! average/sum. Normalization of the weight in the LHEF
-#*********************************************************************
-# Matching parameter (MLM only)
+  True	= gridpack !True = setting up the grid pack
```

Comment:
Just for you info: Setting will be handled internally.

Reply:
Okay, that's fine then, thanks  !